CODEOWNERS

```
@@ -1,4 +1,5 @@
 * @theforeman/packaging
+/plugins/*ansible/ @theforeman/packaging @theforeman/ansible
```

`hammer_cli_foreman_ansible` lives under dependencies

```suggestion
/plugins/*ansible/ @theforeman/packaging @theforeman/ansible
/dependencies/*/*ansible/ @theforeman/packaging @theforeman/ansible
```